src/Tests/Microsoft.Diagnostics.Monitoring.UnitTests/Runners/DotNetMonitorRunner.cs

```diff
@@ -150,24 +157,38 @@ public async Task StartAsync(CancellationToken token)
             _runner.SetEnvironmentVariable("Logging__Console__FormatterName", "json");
             // Enable Debug on Startup class to get lifetime and address events
             _runner.SetEnvironmentVariable("Logging__LogLevel__Microsoft.Diagnostics.Tools.Monitor.Startup", "Debug");
+            
+            // Override the shared config directory
+            _runner.SetEnvironmentVariable("DotnetMonitor_SharedConfigDirectoryOverride", SharedConfigDirectoryPath);
```

Comment:
Can we make sure the metasettings don't look like normal settings? These match our config prefix.

Reply:
Do you have a proposed naming convention? `DotnetMonitor_Testing__`?

Reply:
I would avoid the DotnetMonitor_ prefix, since you don't want this to merge into configuration right? Maybe DotnetMonitorTestSettings_?

Reply:
All environment variables are merged into configuration regardless of the prefix. Configuring with the prefix `DotnetMonitor_` just means "also add these without this prefix". For example, the variables `DotnetMonitor_Metrics__Enabled` and `Metrics__Enabled` are merged into the same key `Metrics:Enabled`.

Reply:
I'll use your suggested prefix `DotnetMonitorTestSettings__` for the variables and not configure this prefix.